solarwinds_apm/api/__init__.py

```diff
@@ -68,7 +75,7 @@ def set_transaction_name(custom_name: str) -> bool:
     entry_trace_id = baggage.get_baggage(INTL_SWO_CURRENT_TRACE_ID)
     entry_span_id = baggage.get_baggage(INTL_SWO_CURRENT_SPAN_ID)
     if not entry_trace_id or not entry_span_id:
-        logger.debug(
+        logger.error(
```

Comment:
I think `warning` might be more appropriate, similar to if the customer passed in an invalid transaction name.

Reply:
Thanks, make sense! Addressed in [32bf45e](https://github.com/solarwindscloud/solarwinds-apm-python/pull/162/commits/32bf45effcc669455986fc6cb3a672972d6841d6)